### PR TITLE
Use GitHub Actions to test and bench

### PR DESCRIPTION
**Pull request by JmPotato**
Signed-off-by: JmPotato <ghzpotato@gmail.com>

You can check some examples from here: https://github.com/JmPotato/yatp/actions

### CODE REVIEW

**Comment by JmPotato** [comment]
cc @BusyJay @sticnarf

**Comment by JmPotato** [comment]
It looks like we need to remove Travis from the CI settings of this repo and move to GitHub Actions. BTW, GitHub Actions is way faster than Travis now.

**Review by sticnarf — APPROVED** [review]
Looks good. We may need @BusyJay to configure the repo.

File: .github/workflows/bench.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Install toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.rust_version }}
+          override: true
+      - name: Bench
+        run: cargo bench --all -- --test
```
Comment (BusyJay) [inline .github/workflows/bench.yaml:36]: Isn't it covered by test.yaml already?

Comment (sticnarf) [reply .github/workflows/bench.yaml:36]: No. `cargo test` does not run benchmarks as tests.

**Review by BusyJay — APPROVED** [review]

**Pull request merged**